### ISSUE TITLE
add a proper settings page

### ISSUE THREAD

**Issue by plexiondev** [plexiondev/plexiondev.github.io#35]
as appose to the accessibility page, would contain more general options for usability, theming, github-api related stuff (👀), and more. (would also link into GSOT upcoming ideas 👀)

unsure on how it should be accessed as currently accessibility is linked in the top-right, possibly similar to pre-rebuild's account menu.

**Comment by plexiondev** [comment]
will also need to consider mobile this time around

pre-rebuild a simple link was just added to the mobile navbar that directly linked to the settings page (without the menu ever appearing)

**Comment by plexiondev** [comment]
pre-rebuild ref.

menu:
![image](https://user-images.githubusercontent.com/46572320/163692480-522175d2-247e-40f3-9c91-9e8451e89d9c.png)

mobile navbar, link:
![image](https://user-images.githubusercontent.com/46572320/163692498-bc2527ca-f3fc-4190-a69a-2385d082ede5.png)
